features/content_lists/add_article_to_manual_content_list_when_published.feature

```diff
@@ -0,0 +1,81 @@
+@disable-fixtures
+@content_lists
+Feature: Add article to manual content list when published
+  In order to add article to content list when it's published
+  As a HTTP Client
+  I want to be able to publish article and see it in content lists
+
+  Scenario: Push new article and add it to automatic content lists
```

Comment:
it should be manual, not automatic because later you create a manual content list